src/mpl/include/mpl_trmem.h

```diff
@@ -19,17 +19,48 @@ extern char *strdup(const char *);
 char *MPL_strdup(const char *str);
 #endif /* defined(MPL_USE_MEMORY_TRACING) || defined(MPL_HAVE_STRDUP) */
 
+typedef enum {
+    MPL_MEM_ADDRESS,        /* Address information */
```

Comment:
Let's rename this to PMI, since not all PMI data is necessarily address info.

Reply:
Fine with me.